gradle/missing-javadoc.gradle

```diff
@@ -0,0 +1,261 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import javax.annotation.Nullable
+import org.gradle.api.tasks.PathSensitive;
+import org.gradle.api.tasks.PathSensitivity;
+import org.gradle.internal.jvm.Jvm;
+
+/**
+ * Checks for missing javadocs.
+ */
+// Original version of this file is ported from the render-javadoc code in Apache Solr,
+// original code under the Apache Software Foundation under Apache 2.0 license
+// See - https://github.com/apache/solr/blob/main/gradle/documentation/render-javadoc.gradle
+
+allprojects {
+  ext {
+    scriptResources = { buildscript ->
+      return file(buildscript.sourceFile.absolutePath.replaceAll('.gradle$', ""))
+    }
+  }
+}
+
+def resources = scriptResources(buildscript)
+
+allprojects {
+  plugins.withType(JavaPlugin) {
+    configurations {
+      missingdoclet
+    }
+
+    dependencies {
+      missingdoclet "org.opensearch:missing-doclet"
```

Comment:
I think the current build failure is because I didn't update this package to .sdk and didn't notice because I had the previous dependency cached.